app/templates/components/public/event-map.hbs

```diff
@@ -1,13 +1,17 @@
 {{#if event.isMapShown}}
   <div class="eight wide column event-map">
     <h1>{{t 'Getting Here'}}</h1>
-    {{#g-map markersFitMode='live' lat=37.744 lng=-122.4367 address=event.locationName zoom=2 class='google-maps' as |context|}}
-      {{#g-map-address-marker context address=event.locationName as |markerContext|}}
-        {{#g-map-infowindow markerContext}}
-          {{event.locationName}}
-        {{/g-map-infowindow}}
-      {{/g-map-address-marker}}
-    {{/g-map}}
+    {{#if (eq mapConfig.display 'embed') }}
+      <iframe class="g-map" src="https://maps.google.com/maps?q={{event.locationName}}&t=&z=15&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no"></iframe>
```

Comment:
Is it temporary @iamareebjamal ,as we are not using any custom calls in the app? If yes, we should just add a comment untill mapbox is implemented, as it will break the coding conventions getting followed for handlebars in the repository.

Reply:
No, it is not temporary. If there is neither mapbox nor google API, this will be rendered. As both services are paid and this is free.

> as we are not using any custom calls in the app?

What do you mean by custom calls? Google Maps API is a custom library and we are using it anyway

> as it will break the coding conventions getting followed for handlebars in the repository

What coding conventions for handlebars, can you elaborate?

Reply:
Nothing,I didnt saw the iframe tag thought it is js tweak.
It will be better to add some text between iframe tag, if iframe is not supported in any device?

Reply:
If iframe is not supported in that device, then the device probably does not even have a working browser

Reply:
😅 Its lgtm